aten/src/ATen/native/cuda/MultiTensorApply.cuh

```diff
@@ -143,8 +157,8 @@ void multi_tensor_apply(
     // per tensor since the zero-sized ones will not enter the loop, so
     // the nested forloop within represents iterating through the chunks
     // of a single tensor.
-    const auto chunks =
-        (tensor_lists[0][t].numel() + kChunkSize - 1) / kChunkSize;
+    const auto numel = tensor_lists[0][t].numel();
+    const auto chunks = numel / kChunkSize + (numel % kChunkSize != 0);
```

Comment:
numel is int64_t so chunks is also, right?

Reply:
numel is a size_t i believe? this change is to prevent overflow (even tho overflow is likely unlikely here)